test/integration/formatting_stack/strategies.clj

```diff
@@ -235,3 +238,18 @@
         (sh "rm" (str rename-destination))
         (assert (-> deletable-file .exists))
         (assert (not (-> rename-destination .exists)))))))
+
+(deftest namespaces-within-refresh-dirs-only
+  (speced/let [^{::speced/spec (rcomp count (partial < 100))}
+               all-files (sut/all-files :files [])
+               result (sut/namespaces-within-refresh-dirs-only :files all-files)]
+    (is (seq result)
+        "Returns non-empty results (since f-s itself has namespaces within `src`, `test`, etc)")
+
+    (is (< (count result)
+           (count all-files))
+        "Doesn't include files outside the refresh dirs")
```

Comment:
Relying on count for this assertion seems error prone. Paired with the next `is` (which does the described assertion), it's fine.

Reply:
Note that any set _x_ is a subset of itself, so a count assertion is essential.

Reply:
For sure. I was mainly targeting the description in this case.

The assertion is only testing of the result is smaller now. The next assertion asserts that it's a subset. Which is👌🏼 when combined.